TelCo.ColorCoder/PairNumberGetter.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+
+namespace TelCo.ColorCoder
+{
+    class PairNumberGetter
+    {
+        /// <summary>
+        /// Given the two colors the function returns the pair number corresponding to them
+        /// </summary>
+        /// <param name="pair">Color pair with major and minor color</param>
+        public static int GetPairNumberFromColor(ColorPair pair)
+        {
+            // Find the major color in the array and get the index
+            int majorIndex = -1;
+            for (int i = 0; i < Program.colorMapMajor.Length; i++)
+            {
+                if (Program.colorMapMajor[i] == pair.majorColor)
+                {
+                    majorIndex = i;
+                    break;
+                }
+            }
+
+            // Find the minor color in the array and get the index
+            int minorIndex = -1;
+            for (int i = 0; i < Program.colorMapMinor.Length; i++)
```

Comment:
This `for` loop is almost a duplicate of the previous one. Any way to remove the duplication?